Transmission.xcodeproj/project.pbxproj

```diff
@@ -1379,96 +1379,96 @@
 		080E96DDFE201D6D7F000001 /* Sources */ = {
 			isa = PBXGroup;
 			children = (
-				29B97316FDCFA39411CA2CEA /* main.mm */,
```

Comment:
I don't think any of these changes here are needed.

Reply:
I agree: @Harm133 please rollback the changes to project.pbxproj. I understand you did right-click "Sort by Name", but that would need a separate discussion and pull request, since it's not obvious that mixing folders with files, or moving `main` in the middle of the list, is the best desired organisation. It might be, but I prefer to have it discussed separately.

Reply:
This was not my intention :) ! I thought the changes were related to files being updated, but it seems I should've looked into this diff a bit more. Reverted.